### ISSUE TITLE
Error Creating a Report Executor

### ISSUE THREAD

**Issue by akranjec** [Adobe-Consulting-Services/acs-aem-commons#1441]
## Required Information

* [x] AEM Version, including Service Packs, Cumulative Fix Packs, etc: AEM 6.4
* [x] ACS AEM Commons Version: 3.17.0
* [x] Reproducible on Latest? yes

### Expected Behavior

According to documentation on Extending Reports it should be possible to create a custom Report Executor.

### Actual Behavior

Custom Report Executor can't be created.

### Steps to Reproduce

1. I used fresh 6.4 AEM instance and set up a fresh project based on AEM archetype 13.
2. Created TestReportExecutor.java class with `@Model` annotation and implemented ReportExecutor interface. 
3. Also, I created a component which uses TestReportExecutor
```
                <reportExecutor
                        jcr:primaryType="nt:unstructured"
                        sling:resourceType="granite/ui/components/foundation/form/hidden"
                        name="./reportExecutor"
                        value="clean.aem.core.models.TestReportExecutor" />
```
4. After using TestReportExecutor in newly created report I recive folowing error in error.log

```
> java.lang.ClassNotFoundException: clean.aem.core.models.TestReportExecutor not found by com.adobe.acs.acs-aem-commons-bundle [546]
> 	at org.apache.felix.framework.BundleWiringImpl.findClassOrResourceByDelegation(BundleWiringImpl.java:1639)
> 	at org.apache.felix.framework.BundleWiringImpl.access$200(BundleWiringImpl.java:80)
> 	at org.apache.felix.framework.BundleWiringImpl$BundleClassLoader.loadClass(BundleWiringImpl.java:2053)
> 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
> 	at com.adobe.acs.commons.reports.models.ReportRunner.executeConfig(ReportRunner.java:66) [com.adobe.acs.acs-aem-commons-bundle:3.17.0]
> 	at com.adobe.acs.commons.reports.models.ReportRunner.init(ReportRunner.java:118) [com.adobe.acs.acs-aem-commons-bundle:3.17.0]
> 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> 	at java.lang.reflect.Method.invoke(Method.java:498)
> 	at org.apache.sling.models.impl.ModelAdapterFactory.invokePostConstruct(ModelAdapterFactory.java:896) [org.apache.sling.models.impl:1.4.7.T20180205124646-b0647a3]
> 	at org.apache.sling.models.impl.ModelAdapterFactory.createObject(ModelAdapterFactory.java:703) [org.apache.sling.models.impl:1.4.7.T20180205124646-b0647a3]
> 	at org.apache.sling.models.impl.ModelAdapterFactory.internalCreateModel(ModelAdapterFactory.java:394) [org.apache.sling.models.impl:1.4.7.T20180205124646-b0647a3]
> 	at org.apache.sling.models.impl.ModelAdapterFactory.getAdapter(ModelAdapterFactory.java:249) [org.apache.sling.models.impl:1.4.7.T20180205124646-b0647a3]
> 	at org.apache.sling.adapter.internal.AdapterManagerImpl.getAdapter(AdapterManagerImpl.java:143) [org.apache.sling.adapter:2.1.10]
> 	at org.apache.sling.engine.impl.SlingMainServlet.adaptTo(SlingMainServlet.java:604) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.engine.impl.request.RequestData.adaptTo(RequestData.java:658) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.engine.impl.SlingHttpServletRequestImpl.adaptTo(SlingHttpServletRequestImpl.java:89) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.api.wrappers.SlingHttpServletRequestWrapper.adaptTo(SlingHttpServletRequestWrapper.java:148) [org.apache.sling.api:2.16.4]
> 	at org.apache.sling.api.wrappers.SlingHttpServletRequestWrapper.adaptTo(SlingHttpServletRequestWrapper.java:148) [org.apache.sling.api:2.16.4]
> 	at org.apache.sling.api.wrappers.SlingHttpServletRequestWrapper.adaptTo(SlingHttpServletRequestWrapper.java:148) [org.apache.sling.api:2.16.4]
> 	at org.apache.sling.scripting.jsp.taglib.AdaptToTag.doEndTag(AdaptToTag.java:56) [org.apache.sling.scripting.jsp.taglib:2.2.6]
> 	at org.apache.jsp.apps.acs_002dcommons.components.utilities.report_002dbuilder.report_002dpage.results_jsp._jspService(results_jsp.java:205)
> 	at org.apache.sling.scripting.jsp.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70) [org.apache.sling.scripting.jsp:2.2.7.B002]
> 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:725) [org.apache.felix.http.servlet-api:1.1.2]
> 	at org.apache.sling.scripting.jsp.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:502) [org.apache.sling.scripting.jsp:2.2.7.B002]
> 	at org.apache.sling.scripting.jsp.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:449) [org.apache.sling.scripting.jsp:2.2.7.B002]
> 	at org.apache.sling.scripting.jsp.JspScriptEngineFactory.callJsp(JspScriptEngineFactory.java:346) [org.apache.sling.scripting.jsp:2.2.7.B002]
> 	at org.apache.sling.scripting.jsp.JspScriptEngineFactory.access$100(JspScriptEngineFactory.java:101) [org.apache.sling.scripting.jsp:2.2.7.B002]
> 	at org.apache.sling.scripting.jsp.JspScriptEngineFactory$JspScriptEngine.eval(JspScriptEngineFactory.java:607) [org.apache.sling.scripting.jsp:2.2.7.B002]
> 	at org.apache.sling.scripting.core.impl.DefaultSlingScript.call(DefaultSlingScript.java:388) [org.apache.sling.scripting.core:2.0.54]
> 	at org.apache.sling.scripting.core.impl.DefaultSlingScript.eval(DefaultSlingScript.java:184) [org.apache.sling.scripting.core:2.0.54]
> 	at org.apache.sling.scripting.core.impl.DefaultSlingScript.service(DefaultSlingScript.java:491) [org.apache.sling.scripting.core:2.0.54]
> 	at org.apache.sling.engine.impl.request.RequestData.service(RequestData.java:552) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.engine.impl.filter.SlingComponentFilterChain.render(SlingComponentFilterChain.java:44) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:77) [org.apache.sling.engine:2.6.12]
> 	at com.day.cq.wcm.core.impl.WCMDebugFilter.doFilterWithErrorHandling(WCMDebugFilter.java:192) [com.day.cq.wcm.cq-wcm-core:5.11.218]
> 	at com.day.cq.wcm.core.impl.WCMDebugFilter.doFilter(WCMDebugFilter.java:159) [com.day.cq.wcm.cq-wcm-core:5.11.218]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.day.cq.wcm.core.impl.WCMComponentFilter.filterRootInclude(WCMComponentFilter.java:375) [com.day.cq.wcm.cq-wcm-core:5.11.218]
> 	at com.day.cq.wcm.core.impl.WCMComponentFilter.doFilter(WCMComponentFilter.java:190) [com.day.cq.wcm.cq-wcm-core:5.11.218]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.day.cq.wcm.core.impl.PageLockFilter.doFilter(PageLockFilter.java:91) [com.day.cq.wcm.cq-wcm-core:5.11.218]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.day.cq.personalization.impl.TargetComponentFilter.doFilter(TargetComponentFilter.java:96) [com.day.cq.cq-personalization:5.11.54]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.engine.impl.SlingRequestProcessorImpl.processComponent(SlingRequestProcessorImpl.java:282) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.engine.impl.filter.RequestSlingFilterChain.render(RequestSlingFilterChain.java:49) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:71) [org.apache.sling.engine:2.6.12]
> 	at com.day.cq.wcm.core.impl.warp.TimeWarpFilter.doFilter(TimeWarpFilter.java:109) [com.day.cq.wcm.cq-wcm-core:5.11.218]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.adobe.cq.social.ugcbase.security.impl.SaferSlingPostServlet.doFilter(SaferSlingPostServlet.java:114) [com.adobe.cq.social.cq-social-ugcbase-impl:2.4.19]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:73) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:73) [org.apache.sling.engine:2.6.12]
> 	at com.day.cq.dam.core.impl.servlet.ActivityRecordHandler.doFilter(ActivityRecordHandler.java:154) [com.day.cq.dam.cq-dam-core:5.11.296]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.day.cq.dam.core.impl.assetlinkshare.AdhocAssetShareAuthHandler.doFilter(AdhocAssetShareAuthHandler.java:437) [com.day.cq.dam.cq-dam-core:5.11.296]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.adobe.granite.rest.impl.servlet.ApiResourceFilter.doFilter(ApiResourceFilter.java:70) [com.adobe.granite.rest.api:1.1.10]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.adobe.granite.requests.logging.impl.RequestLoggerImpl.doFilter(RequestLoggerImpl.java:134) [com.adobe.granite.requests.logging:1.0.20]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.adobe.granite.rest.assets.impl.AssetContentDispositionFilter.doFilter(AssetContentDispositionFilter.java:96) [com.adobe.granite.rest.assets:1.0.56]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.adobe.granite.csrf.impl.CSRFFilter.doFilter(CSRFFilter.java:217) [com.adobe.granite.csrf:1.0.18]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.day.cq.wcm.core.impl.AuthoringUIModeServiceImpl.doFilter(AuthoringUIModeServiceImpl.java:297) [com.day.cq.wcm.cq-wcm-core:5.11.218]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.day.cq.wcm.mobile.core.impl.redirect.RedirectFilter.doFilter(RedirectFilter.java:248) [com.day.cq.wcm.cq-wcm-mobile-core:5.11.10]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at clean.aem.core.filters.LoggingFilter.doFilter(LoggingFilter.java:56) [clean.aem.cms.core:1.0.0.SNAPSHOT]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.engine.impl.debug.RequestProgressTrackerLogFilter.doFilter(RequestProgressTrackerLogFilter.java:107) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:73) [org.apache.sling.engine:2.6.12]
> 	at com.adobe.cq.social.commons.cors.CORSAuthenticationFilter.doFilter(CORSAuthenticationFilter.java:91) [com.adobe.cq.social.cq-social-commons:1.9.47]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.day.cq.wcm.foundation.forms.FormsHandlingServletHelper.handleFilter(FormsHandlingServletHelper.java:226) [com.day.cq.wcm.cq-wcm-foundation:5.11.46]
> 	at com.day.cq.wcm.foundation.forms.impl.FormsHandlingServlet.doFilter(FormsHandlingServlet.java:138) [com.day.cq.wcm.cq-wcm-foundation:5.11.46]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.adobe.granite.optout.impl.OptOutFilter.doFilter(OptOutFilter.java:76) [com.adobe.granite.optout:1.0.0]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.day.cq.wcm.foundation.forms.FormsHandlingServletHelper.handleFilter(FormsHandlingServletHelper.java:226) [com.day.cq.wcm.cq-wcm-foundation:5.11.46]
> 	at com.adobe.cq.wcm.core.components.internal.servlets.CoreFormHandlingServlet.doFilter(CoreFormHandlingServlet.java:126) [com.adobe.cq.core.wcm.components.core:2.0.4]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.day.cq.wcm.core.impl.WCMRequestFilter.doFilter(WCMRequestFilter.java:90) [com.day.cq.wcm.cq-wcm-core:5.11.218]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.adobe.cq.history.impl.HistoryRequestFilter.doFilter(HistoryRequestFilter.java:122) [com.adobe.cq.cq-history:5.10.6]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.adobe.cq.mcm.campaign.servlets.CampaignCopyTracker.doFilter(CampaignCopyTracker.java:100) [com.day.cq.mcm.cq-mcm-campaign:1.3.8]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.rewriter.impl.RewriterFilter.doFilter(RewriterFilter.java:87) [org.apache.sling.rewriter:1.2.2]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at com.adobe.granite.httpcache.impl.InnerCacheFilter.doFilter(InnerCacheFilter.java:81) [com.adobe.granite.httpcache.core:1.0.8]
> 	at com.adobe.granite.httpcache.impl.InnerCacheFilter.doFilter(InnerCacheFilter.java:60) [com.adobe.granite.httpcache.core:1.0.8]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.i18n.impl.I18NFilter.doFilter(I18NFilter.java:131) [org.apache.sling.i18n:2.5.10]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.security.impl.ContentDispositionFilter.doFilter(ContentDispositionFilter.java:152) [org.apache.sling.security:1.1.10]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:73) [org.apache.sling.engine:2.6.12]
> 	at com.adobe.granite.resourceresolverhelper.impl.ResourceResolverHelperImpl.doFilter(ResourceResolverHelperImpl.java:83) [com.adobe.granite.resourceresolverhelper:1.0.4]
> 	at org.apache.sling.engine.impl.filter.AbstractSlingFilterChain.doFilter(AbstractSlingFilterChain.java:68) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.engine.impl.SlingRequestProcessorImpl.doProcessRequest(SlingRequestProcessorImpl.java:151) [org.apache.sling.engine:2.6.12]
> 	at org.apache.sling.engine.impl.SlingMainServlet.service(SlingMainServlet.java:234) [org.apache.sling.engine:2.6.12]
> 	at org.apache.felix.http.base.internal.handler.ServletHandler.handle(ServletHandler.java:120) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.felix.http.base.internal.dispatch.InvocationChain.doFilter(InvocationChain.java:86) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.sling.security.impl.ReferrerFilter.doFilter(ReferrerFilter.java:328) [org.apache.sling.security:1.1.10]
> 	at org.apache.felix.http.base.internal.handler.FilterHandler.handle(FilterHandler.java:135) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.felix.http.base.internal.dispatch.InvocationChain.doFilter(InvocationChain.java:81) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at com.adobe.granite.license.impl.LicenseCheckFilter.doFilter(LicenseCheckFilter.java:308) [com.adobe.granite.license:1.2.6.CQ640-B0001]
> 	at org.apache.felix.http.base.internal.handler.FilterHandler.handle(FilterHandler.java:135) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.felix.http.base.internal.dispatch.InvocationChain.doFilter(InvocationChain.java:81) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.felix.http.sslfilter.internal.SslFilter.doFilter(SslFilter.java:96) [org.apache.felix.http.sslfilter:1.2.4]
> 	at org.apache.felix.http.base.internal.handler.FilterHandler.handle(FilterHandler.java:135) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.felix.http.base.internal.dispatch.InvocationChain.doFilter(InvocationChain.java:81) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.sling.i18n.impl.I18NFilter.doFilter(I18NFilter.java:131) [org.apache.sling.i18n:2.5.10]
> 	at org.apache.felix.http.base.internal.handler.FilterHandler.handle(FilterHandler.java:135) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.felix.http.base.internal.dispatch.InvocationChain.doFilter(InvocationChain.java:81) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.sling.featureflags.impl.FeatureManager.doFilter(FeatureManager.java:116) [org.apache.sling.featureflags:1.2.2]
> 	at org.apache.felix.http.base.internal.handler.FilterHandler.handle(FilterHandler.java:135) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.felix.http.base.internal.dispatch.InvocationChain.doFilter(InvocationChain.java:81) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.sling.engine.impl.log.RequestLoggerFilter.doFilter(RequestLoggerFilter.java:72) [org.apache.sling.engine:2.6.12]
> 	at org.apache.felix.http.base.internal.handler.FilterHandler.handle(FilterHandler.java:135) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.felix.http.base.internal.dispatch.InvocationChain.doFilter(InvocationChain.java:81) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.sling.engine.impl.parameters.RequestParameterSupportConfigurer.doFilter(RequestParameterSupportConfigurer.java:63) [org.apache.sling.engine:2.6.12]
> 	at org.apache.felix.http.base.internal.handler.FilterHandler.handle(FilterHandler.java:135) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.felix.http.base.internal.dispatch.InvocationChain.doFilter(InvocationChain.java:81) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.felix.http.base.internal.dispatch.Dispatcher$1.doFilter(Dispatcher.java:147) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.felix.http.base.internal.whiteboard.WhiteboardManager.invokePreprocessors(WhiteboardManager.java:1000) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.felix.http.base.internal.dispatch.Dispatcher.dispatch(Dispatcher.java:91) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.apache.felix.http.base.internal.dispatch.DispatcherServlet.service(DispatcherServlet.java:49) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:725) [org.apache.felix.http.servlet-api:1.1.2]
> 	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:848) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:584) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:224) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1180) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:512) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1112) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:213) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:134) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.server.Server.handle(Server.java:534) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:333) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:251) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:283) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:108) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.io.SelectChannelEndPoint$2.run(SelectChannelEndPoint.java:93) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.executeProduceConsume(ExecuteProduceConsume.java:303) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.produceConsume(ExecuteProduceConsume.java:148) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.run(ExecuteProduceConsume.java:136) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:671) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:589) [org.apache.felix.http.jetty:3.4.7.B012]
> 	at java.lang.Thread.run(Thread.java:748)
```

### Links

_Documentation on Extending Reports
https://adobe-consulting-services.github.io/acs-aem-commons/features/report-builder/extending.html#creating-a-report-executor_

**Comment by davidjgonzalez** [comment]
@akranjec i assume youve confirmed that your Sling Model is infact registered and using something like AEM Fiddle (ACS Tools) you can adapt a Sling Request to your Model class, correct?

**Comment by davidjgonzalez** [comment]
@akranjec  can you try out this package on your dev env and see if it fixes it?

[acs-aem-commons-content-3.17.1-SNAPSHOT-min.zip](https://github.com/Adobe-Consulting-Services/acs-aem-commons/files/2249861/acs-aem-commons-content-3.17.1-SNAPSHOT-min.zip)

**Comment by akranjec** [comment]
Yep, now it works

**Comment by akranjec** [comment]
@davidjgonzalez thank you very much. When will this be released?

**Comment by davidjgonzalez** [comment]
Fixed in 3.17.2
